src/services/gateways/aws.ts

```diff
@@ -210,21 +210,30 @@ export class AWS {
     return role.Role?.Arn ?? '';
   }
 
+  async newRoleLin(
```

Comment:
maybe we should delete the non-lin methods?

Reply:
That would change the behavior of the prior module versions.

Reply:
Oh, also, I am hoping to follow-up this PR with one that eliminates the AWS gateway file from usage in the `0.0.3` modules, so this would be rolled back in that case.